ODPC.Server/Data/OdpcDbContext.cs

```diff
@@ -10,19 +11,36 @@ public OdpcDbContext(DbContextOptions options) : base(options)
         }
 
         public DbSet<WeatherForecast> WeatherForecasts { get; set; }
+        public DbSet<Gebruikersgroep> Gebruikersgroepen { get; set; }
+        public DbSet<GebruikersgroepWaardelijst> GebruikersgroepWaardelijsten { get; set; }
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             base.OnModelCreating(modelBuilder);
 
-            // Seed data
+
+            modelBuilder.Entity<Gebruikersgroep>().HasKey(t => new { t.Id });
+            modelBuilder.Entity<GebruikersgroepWaardelijst>().HasKey(t => new { t.GebruikersgroepId, t.WaardelijstId });
+
+
+            // Seed data 
+            // todo verwijderen
             modelBuilder.Entity<WeatherForecast>().HasData(
```

Comment:
Wat mij betreft dit zo snel mogelijk weghalen, hierdoor komen er bij elke migratie onnodig extra records in de database!